src/riscv/lib/src/pvm/linux.rs

```diff
@@ -898,6 +902,50 @@ impl<M: ManagerBase> SupervisorState<M> {
         Ok(0)
     }
 
+    /// Handle `sched_getaffinity` system call. We only support one hart.
+    ///
+    /// See: <https://man7.org/linux/man-pages/man2/sched_getaffinity.2.html>
+    fn handle_sched_getaffinity(
+        &self,
+        core: &mut MachineCoreState<impl MemoryConfig, M>,
+        pid: parameters::NullableProcessId,
+        cpusetsize: parameters::CpuSetSize,
+        mask: parameters::CpuSet,
```

Comment:
This could be `VirtAddr` instead. I don't see what `CpuSet` adds here as a type.